bbc/simorgh
# Fix readme links

Resolves JIRA [number]

Overall changes
======
Updated/fixed links to documentation. No files have been renamed, just fixing links to now non-existent files.

Noticed this as I was writing up tickets for the amp experiment work, figured it would be worth the quick cleanup for anyone else needing to read through the docs.

@HarveyPeachey mentioned this PR (https://github.com/bbc/simorgh/pull/11877), so I'm assuming the links _should_ now be `.mdx` and not `.stories.mdx`. Feel free to ignore and close this PR if I've got it the wrong way round!

Code changes
======
- Changed links leading to documentation files ending with `.stories.mdx` to `.mdx`

Testing
======
1. Open links in top level readme to ensure that they work as expected.
2. Open storybook locally and ensure that the documentation section still works correctly.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
